Src/CSharpier/SyntaxPrinter/Assignment.cs

```diff
@@ -0,0 +1,69 @@
+using System;
+using CSharpier.DocTypes;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+
+namespace CSharpier.SyntaxPrinter
+{
+    public static class Assignment
+    {
+        public static Doc PrintRhs(ExpressionSyntax rhs)
```

Comment:
I'm wondering if this should be named
```
RightHandSide.Print()
```
And I'd probably go with `ExpressionSyntax node` to keep it consistent with other classes. I had to think about what `rhs` was when looking at this.